smoke/IotEdgeQuickstart/Quickstart.cs

```diff
@@ -67,7 +67,6 @@ public async Task RunAsync()
                         await DeployToEdgeDevice();
                         if (!this.noVerify)
                         {
-                            await VerifyTempSensorIsRunning();
```

Comment:
Should we make this check parameterized instead? So that we can check that module X is running?

Reply:
yep. Added a task for that (to be made generic and for multiples modules as well)